Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/weatherInput/index.tsx

```diff
@@ -1,4 +1,9 @@
-const WeatherInput = () => {
+const WeatherInput = (props: {
+  setWeather: React.Dispatch<React.SetStateAction<number>>;
+}) => {
+  const handleWeatherChange = (newValue: number) => {
```

Comment:
You should be controlling your input here from your weather state variable in App.tsx